python/cugraph/tests/dask/opg_pagerank_testing.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cugraph.dask as dcg
+from dask.distributed import Client
+import gc
+import cugraph
+import dask_cudf
+
+# Move to conftest
+from dask_cuda import LocalCUDACluster
+# cluster = LocalCUDACluster(protocol="tcp", scheduler_port=0)
```

Comment:
Should this comment be removed from here?

Reply:
This is a temporary test (which is not run by CI) This is to check the validity of the PR if someone wants to test the pipeline. This is going to be replaced by a pagerank test after #947 is merged and e2e runs successfully.